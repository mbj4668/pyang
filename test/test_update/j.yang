module j {
  yang-version 1.1;
  namespace urn:j;
  prefix j;

  revision 2014-03-01;

  leaf foo {
    type enumeration {
      enum f {
        value 1;
      }
    }
  }
}

