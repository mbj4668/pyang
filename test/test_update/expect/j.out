j@2014-04-01.yang:9: error: CHK_ENUM_VALUE_CHANGED_v1.1
